src/wallet/wallet.cpp

```diff
@@ -6746,11 +6728,7 @@ bool CWallet::ParameterInteraction(const CChainParams& params)
 {
     if (mapArgs.count("-mintxfee"))
     {
-        CAmount n = 0;
-        if (ParseMoney(mapArgs["-mintxfee"], n) && n > 0)
-            CWallet::minTxFee = CFeeRate(n);
-        else
-            return UIError(AmountErrMsg("mintxfee", mapArgs["-mintxfee"]));
+        UIWarning(_("The argument -mintxfee is no longer supported."));
     }
     if (mapArgs.count("-paytxfee"))
```

Comment:
In order for fee rate calculations not to overflow, we must have `nSatoshisPerK <= INT64_MAX / MAX_BLOCK_SIZE`. `ParseMoney` does not ensure that limit on its result (surprisingly to me, it does not even ensure a limit of `MAX_MONEY`).